src/include/version.h

```diff
@@ -0,0 +1,110 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WT_VERSION --
+ *	Structure to represent version information.
+ */
+struct __wt_version {
+    uint16_t major;
+    uint16_t minor;
+    uint16_t patch;
+};
+
+/*
+ * WT_BTREE_VERSION is an alias of WT_VERSION to avoid confusion between btree version numbers and
+ * WiredTiger version numbers.
+ */
+typedef WT_VERSION WT_BTREE_VERSION;
+
+#define WT_NO_VALUE UINT16_MAX
+/*
+ * Default version to use when none is defined.
+ */
+#define WT_NO_VERSION ((WT_VERSION){WT_NO_VALUE, WT_NO_VALUE, WT_NO_VALUE})
+
+/*
+ * __wt_version_cmp --
+ *     Compare two versions to determine whether the first version is greater than, equal to, or
+ *     less than the second. As in strcmp() return 1 for greater, 0 for equal, and -1 for less than.
+ */
+static inline int32_t
+__wt_version_cmp(WT_VERSION v, WT_VERSION other)
+{
+    /*
+     * The patch version is not always set for both inputs. In these cases we ignore comparison of
+     * patch version by setting them both to the same value. The inputs are pass-by-value and will
+     * not be modified by this.
+     */
+    if (v.patch == WT_NO_VALUE || other.patch == WT_NO_VALUE) {
```

Comment:
We usually dont have curly brackets for one line if statement

Reply:
Fixed in cea28f3